Add comparastore as a company that uses Turbo

angeblecon opened the pull request:
Very grateful for the work of the developers behind this incredible tool that is Turbo. 🙏 As a company that implemented Turborepo and very soon Turbopack, we would be honoured to have our logo on the showcase page.

nathanhammond reviewed (changes requested):
Hey! I appreciate this, but [the site](https://www.comparastore.com/) is throwing a 403 for me (I'm in Hong Kong, in case you've location-restricted things). Addressing that is a prerequisite for being included.

```
403 ERROR
The request could not be satisfied.
Request blocked. We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner.
If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation.

Generated by cloudfront (CloudFront)
Request ID: Y5J7Ey-kYYqaE7QIXb7PreM8LsBF37UmP_e3py8C-XOzL8IbWAvpOw==
```

Also, if you have a different logo for light mode which doesn't include the black block background that would be _much_ preferred.

angeblecon commented:
Thank you for your response, indeed, there was a restriction for Honk Kong that I just removed, you should be able to access the platform now! Also, I just updated the logo to remove the black box. Let me know if you need anything else, thanks!

angeblecon commented:
Thank you very much! Very appreciated
